dockerfile/avalanche/Dockerfile

```diff
@@ -117,16 +114,15 @@ LABEL org.opencontainers.image.source="https://github.com/strangelove-ventures/h
 
 WORKDIR /bin
 
-# Install ln (for making hard links), rm (for cleanup), mv, mkdir amd vi from full busybox image (will be deleted, only needed for image assembly)
-COPY --from=busybox-full /bin/ln /bin/rm /bin/mv /bin/mkdir /bin/vi ./
-
 # Install minimal busybox image as shell binary (will create hardlinks for the rest of the binaries to this data)
 COPY --from=infra-toolkit /busybox/busybox /bin/sh
 
+COPY --from=infra-toolkit /busybox/busybox /bin/ln
```

Comment:
This should replace the copy to `sh`, not be in addition

Reply:
we should keep the comment but update it from `shell` to `ln`, as it helps understand what's going on here.